src/constants/ProjectConstants.ts

```diff
@@ -0,0 +1,11 @@
+export enum VOTE_DECISION {
+  APPROVE = "Approve",
+  REJECT = "Reject",
+  ABSTAIN = "Abstain",
+}
+
+export enum MATTER_STATUS_DECISION {
+  ADOPTED = "Adopted",
+  REJECTED = "Rejected",
+  IN_PROGRESS = "In Progress",
+}
```

Comment:
Could we rename this file from `ProjectConstants.ts` to something like `VotingTableConstants.ts`? Just to make it easier to get an idea of what the file contains without having to open it.

Reply:
I was thinking this would grow over time, gradually encompassing all the various enums that we expect to be used.  Perhaps splitting it out into constants related to votes/other model objects as well? Basically this is a temporary spot, I was thinking we'd split is as we come to know more about the project structure.